test/base/TextTest.ooc

```diff
@@ -226,6 +226,17 @@ TextTest: class extends Fixture {
 			text = t"%s %s %i %i" format("test", "number", 0, 1)
 			expect(text == t"test number 0 1")
 		})
+		this add("implicit toText", func {
+			one := t"123" + 456 + 7.89f
```

Comment:
Here's one of the reasons I suggested implicit conversions in the first place:
```ooc
	line: static func ~dashed (start, end: FloatPoint2D, lineWidth: Float, opacity: Float, color: String, dashFloatPair: FloatPoint2D) -> String {
		"<line x1='" << start x toString() >> "' y1='" & start y toString() >> "' x2='" & end x toString() >> "' y2='" & end y toString() >> "' stroke-width='" & lineWidth toString() >> "' stroke='" >> color >> "' stroke-opacity='" & opacity toString() >> "' stroke-dasharray='" & dashFloatPair toString() >> "'/>\n"
	}

```
@sebastianbaginski I think the implicit conversion it's clear enough, but I also think you have a point. Another way is to use a varargs concatenation function that accepts arguments of any type and builds a longer string from them. In C#,
```cs
return String.Concat(new object[] { "<line x1='", start.x, "' y1="', start.y, '" x2="' /* etc */ });
```
Unfortunately, varargs in ooc give no compile-time guarantee that the objects passed to such a function actually have a `toString()` function, and neither would passing any collection of `T`. Working interfaces would be a godsend here.